.spelling

```diff
@@ -1068,3 +1068,11 @@ Expat-15
 - troubleshooting/known_issues/antero_node_NID_allocation.md
 # To allow Site Init
 Init
+
+- operations/configuration_management/iSCSI_SBPS_Node_Personalization.md
+enablement
+LUNs
+fileio
+cmd
+iscsi-sbps-targets-config
+targetcli
```

Comment:
```suggestion
iscsi-sbps-targets-config
```
The work "cmd" should be written as command.
I believe everywhere that targetcli is used it is within "`" so I don't think it needs to be added to spelling

Reply:
Fixed.